townhall/myapi/tests/test_opportunity.py

```diff
@@ -4,70 +4,17 @@
 from datetime import datetime
 from django.utils import timezone
 
-# Run test class with "python manage.py test"
+# OPPORTUNITY
+
+# Testing ALL tests "python manage.py test myapi.tests"
+# Testing ONLY opportunity "python manage.py test myapi.tests.test_opportunity"
 
 class TownhallTestCase(TestCase):
```

Comment:
The classes of each file can be adjusted to like `TestOpportunityModel` (and its variations)

Reply:
Just updated it with a recent commit!